ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/ovn/provision/cucushift-installer-rehearse-gcp-upi-ovn-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-ovn-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-gcp
```

Comment:
No reference `ovn-conf` ref ?

Reply:
> In this ovn chain, no reference `ovn-conf` ref ?

@jianlinliu Fixed. Thanks for reviewing!